src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+import "./button.css";
+
+const Button = (props) => {
+  return (
+    <div>
+      <a
+        className={`button ${props.color}`}
```

Comment:
from the Figma design, it looks like the font type in buttons is different from the main title. I would set that here so it can be consistent in all of the buttons.